### DISCUSSION TITLE
Add `component:traitUpdate`

### DISCUSSION THREAD

**Discussion by bgrand-ch** [GrapesJS/grapesjs#5664] [Ideas]
Hello,

The `component:styleUpdate` is very useful.

Is it possible to also add the `component:traitUpdate` event please?

**Comment by artf** [comment]
A trait usually updates attributes/props of the component (based on trait `changeProp` value) and you should already be able to listen to those changes, or am I missing something?

**Reply by bgrand-ch** [reply]
The idea is to be able to retrieve in a single place all modifications to the traits of all components, similar to the way `component:styleUpdate` works.

**Reply by artf** [reply]
Yeah makes sense, I actually added this for the next release https://github.com/GrapesJS/grapesjs/blob/fb5175fd1edd13a2db54d661193fb61a354f8625/src/trait_manager/types.ts#L177-L182
